app/metrics.yaml

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+distribution:
+  channel_name:
+    type: string
+    lifetime: application
+    description: >
+      The distribution channel name of this application. We use this field to recognize Firefox Reality
+      is distributed to which channels, such as FirefoxReality_wavevr, FirefoxReality_oculusvr, FirefoxReality_googlevr, etc.
+    send_in_pings:
+      - baseline
+      - events
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1420
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-541260627
```

Comment:
This should be the link to the [response](https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568), not the request, if possible.